utilities/constants/components.py

```
@@ -0,0 +1,274 @@
+    VIRT_API,
+    VIRT_CONTROLLER,
+    VIRT_HANDLER,
+    VIRT_OPERATOR,
+    VIRT_TEMPLATE_VALIDATOR,
+    VIRT_EXPORTPROXY,
+    KUBEVIRT_APISERVER_PROXY,
+    KUBEVIRT_IPAM_CONTROLLER_MANAGER,
+    VIRT_PLATFORM_AUTOPILOT,
+    VIRT_SYNCHRONIZATION_CONTROLLER,
+]
+ALL_CNV_PODS = CNV_PODS_NO_HPP_CSI_HPP_POOL + [HOSTPATH_PROVISIONER_CSI, HPP_POOL]
```

Ruff (0.15.15)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)